ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -1,13 +1,15 @@
 ---
 
-- name: install Rust
+- name: install Rust deps
   apt:
     name:
-      - cargo
       - pkg-config
       - libssl-dev
     state: present
 
+- name: install rustup in userspace for root
+  shell: curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
Can we pre-install rustup for each user by just copying these files or so into /etc/skel, perhaps?

I think it's something we have to do right away, otherwise the files don't get copied after the user is first created.

Reply:
Sorry, going on vacation now, I won't be able to do that. We can always run scripts for all users later if we want to do sth like that.